jmxtrans-output/jmxtrans-output-elastic/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jmxtrans</groupId>
+		<artifactId>jmxtrans-output</artifactId>
+		<version>251-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>jmxtrans-output-elastic</artifactId>
+
+	<name>JmxTrans - ElasticSearch output writers</name>
+
+	<properties>
+		<verify.mutationThreshold>80</verify.mutationThreshold>
```

Comment:
Mutation coverage is still a bit low (you should find the report in `jmxtrans-output/jmxtrans-output-elastic/target/pit-reports`. Ideally, there are a few scenarios that you should add to the tests or just drop the minimum coverage to 55.

Things that probably make sense to be added to tests:

* sending non numeric values
* validate that the json sent is syntactically valid
* move the `createAlias()` method to the `Server` class and test it there (and refactor all the places where we have the same code)
* test that the index is not created if it already exists
* test that exception is thrown if index cannot be created
* ...

I'll be happy to merge even without those additional tests (I already make you work pretty hard on this one).